appveyor.yml

```diff
@@ -1,26 +1,29 @@
+# Warning: This file is automatically synced from https://github.com/ipfs/ci-sync so if you want to change it, please change it there and ask someone to sync all repositories.
+version: "{build}"
+
 environment:
   matrix:
     - nodejs_version: "6"
     - nodejs_version: "8"
 
-init:
-  - git config --global core.autocrlf input
```

Comment:
This is needed to prevent git from changing LF to CRLF.  See https://github.com/ipfs/js-ipfs-repo/pull/153#issuecomment-346320309

Reply:
So this is needed everywhere and https://github.com/ipfs/ci-sync/blob/master/configs/appveyor.yml should be updated

Reply:
@VictorBjelkholm Did some reading on appveyor.  By default `core.autocrlf` is set to `input`.